config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-periodics.yaml

```diff
@@ -1,5 +1,5 @@
 periodics:
-- name: periodic-cluster-api-provider-aws-e2e-master
+- name: periodic-cluster-api-provider-aws-e2e-v1alpha3
```

Comment:
Mass renamed jobs to better indicate what Cluster API version they are applicable to vs the branch name (since `master` is now essentially the v0.5.x/v1alpha3 release branch